12/7/2018 2:27:10 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:28:03 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:28:06 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:28:12 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:29:17 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Service use 
                    EIDSS Database 'EIDSS_TH'
                    EIDSS Archive Database ''
                    AVR Database 'EIDSS_TH_AVR_Service' 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:29:25 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:29:25 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:29:27 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 33844; Name = )
  Extended Properties: 

12/7/2018 2:31:00 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 22716; Name = )
  Extended Properties: 

12/7/2018 2:31:17 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 22716; Name = )
  Extended Properties: 

12/7/2018 2:31:17 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 22716; Name = )
  Extended Properties: 

12/7/2018 2:31:22 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 22716; Name = )
  Extended Properties: 

12/7/2018 2:36:08 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 33464; Name = )
  Extended Properties: 

12/7/2018 2:36:08 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 33464; Name = )
  Extended Properties: 

12/7/2018 2:36:08 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 33464; Name = )
  Extended Properties: 

12/7/2018 2:36:14 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 33464; Name = )
  Extended Properties: 

12/7/2018 2:36:57 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 30872; Name = )
  Extended Properties: 

12/7/2018 2:36:57 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 30872; Name = )
  Extended Properties: 

12/7/2018 2:36:57 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 30872; Name = )
  Extended Properties: 

12/7/2018 2:37:03 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 30872; Name = )
  Extended Properties: 

12/7/2018 2:38:39 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 28824; Name = )
  Extended Properties: 

12/7/2018 2:38:39 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 28824; Name = )
  Extended Properties: 

12/7/2018 2:38:39 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 28824; Name = )
  Extended Properties: 

12/7/2018 2:38:47 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 28824; Name = )
  Extended Properties: 

12/7/2018 2:41:33 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 34208; Name = )
  Extended Properties: 

12/7/2018 2:41:34 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 34208; Name = )
  Extended Properties: 

12/7/2018 2:41:34 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 34208; Name = )
  Extended Properties: 

12/7/2018 2:41:39 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 34208; Name = )
  Extended Properties: 

12/7/2018 2:57:46 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 33156; Name = )
  Extended Properties: 

12/7/2018 2:57:46 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 33156; Name = )
  Extended Properties: 

12/7/2018 2:57:46 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 33156; Name = )
  Extended Properties: 

12/7/2018 2:57:47 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 33156; Name = )
  Extended Properties: 

12/7/2018 3:11:03 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:11:04 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:11:04 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:11:14 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:13:40 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Service use 
                    EIDSS Database 'EIDSS_TH'
                    EIDSS Archive Database ''
                    AVR Database 'EIDSS_TH_AVR_Service' 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:13:40 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:13:40 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:13:40 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 22972; Name = )
  Extended Properties: 

12/7/2018 3:29:00 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:29:00 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:29:00 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:29:06 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:29:27 PM AVR Service could not connect to Actual EIDSS Database. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 10812; Name = )
  Extended Properties: Exception - An exception of type 'eidss.model.AVR.DataBase.AvrDataException' occurred and was caught.
-----------------------------------------------------------------------------------------
12/07/2018 15:29:27
Type : eidss.model.AVR.DataBase.AvrDataException, eidss.core, Version=6.1.0.42, Culture=neutral, PublicKeyToken=null
Message : AVR Service could not connect to Actual EIDSS Database.
Source : eidss.avr.service
Help link : 
Data : System.Collections.ListDictionaryInternal
TargetSite : eidss.model.AVR.DataBase.DatabaseNames GetDatabaseNames()
HResult : -2146232832
Stack Trace :    at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 451
   at EIDSS.AVR.Service.WcfFacade.AVRFacade.GetDatabaseName() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AVRFacade.cs:line 528

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:29:27 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
	Message : Login failed for user 'super'.
	Source : BLToolkit.4
	Help link : 
	Number : 18456
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
	HResult : -2146233087
	Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
	   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
	   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
	   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 445

		Inner Exception
		---------------
		Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
		Message : Login failed for user 'super'.
		Source : .Net SqlClient Data Provider
		Help link : 
		Errors : System.Data.SqlClient.SqlErrorCollection
		ClientConnectionId : c7e38451-7b62-4f2d-ad7a-ea3e17db6f9e
		Class : 14
		LineNumber : 65536
		Number : 18456
		Procedure : 
		Server : 192.255.51.85\NGSQL08QA
		State : 1
		ErrorCode : -2146232060
		Data : System.Collections.ListDictionaryInternal
		TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
		HResult : -2146232060
		Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
		   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
		   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.Open()
		   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:29:27 PM Login failed for user 'super'. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 10812; Name = )
  Extended Properties: Exception - An exception of type 'BLToolkit.Data.DataException' occurred and was caught.
----------------------------------------------------------------------------
12/07/2018 15:29:27
Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
Message : Login failed for user 'super'.
Source : BLToolkit.4
Help link : 
Number : 18456
Data : System.Collections.ListDictionaryInternal
TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
HResult : -2146233087
Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 445

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:29:27 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
	Message : Login failed for user 'super'.
	Source : .Net SqlClient Data Provider
	Help link : 
	Errors : System.Data.SqlClient.SqlErrorCollection
	ClientConnectionId : c7e38451-7b62-4f2d-ad7a-ea3e17db6f9e
	Class : 14
	LineNumber : 65536
	Number : 18456
	Procedure : 
	Server : 192.255.51.85\NGSQL08QA
	State : 1
	ErrorCode : -2146232060
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
	HResult : -2146232060
	Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
	   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
	   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
	   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
	   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
	   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
	   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
	   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
	   at System.Data.SqlClient.SqlConnection.Open()
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:29:27 PM Login failed for user 'super'. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 10812; Name = )
  Extended Properties: Exception - An exception of type 'System.Data.SqlClient.SqlException' occurred and was caught.
----------------------------------------------------------------------------------
12/07/2018 15:29:27
Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
Message : Login failed for user 'super'.
Source : .Net SqlClient Data Provider
Help link : 
Errors : System.Data.SqlClient.SqlErrorCollection
ClientConnectionId : c7e38451-7b62-4f2d-ad7a-ea3e17db6f9e
Class : 14
LineNumber : 65536
Number : 18456
Procedure : 
Server : 192.255.51.85\NGSQL08QA
State : 1
ErrorCode : -2146232060
Data : System.Collections.ListDictionaryInternal
TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
HResult : -2146232060
Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:29:27 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing


12/7/2018 3:29:29 PM AVR Service could not connect to Actual EIDSS Database. (EIDSS.AVR  Error) 
  Message: AVR Service could not connect to Actual EIDSS Database. 
  Thread: (ID = 10812; Name = )
  Extended Properties: Exception - An exception of type 'eidss.model.AVR.DataBase.AvrDataException' occurred and was caught.
-----------------------------------------------------------------------------------------
12/07/2018 15:29:28
Type : eidss.model.AVR.DataBase.AvrDataException, eidss.core, Version=6.1.0.42, Culture=neutral, PublicKeyToken=null
Message : AVR Service could not connect to Actual EIDSS Database.
Source : eidss.avr.service
Help link : 
Data : System.Collections.ListDictionaryInternal
TargetSite : eidss.model.AVR.DataBase.DatabaseNames GetDatabaseNames()
HResult : -2146232832
Stack Trace :    at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 451
   at EIDSS.AVR.Service.WcfFacade.AVRFacade.GetDatabaseName() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AVRFacade.cs:line 533
   at EIDSS.AVR.Service.WcfService.AvrHostKeeper.InitServiceHost() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfService\AvrHostKeeper.cs:line 57

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:29:29 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
	Message : Login failed for user 'super'.
	Source : BLToolkit.4
	Help link : 
	Number : 18456
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
	HResult : -2146233087
	Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
	   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
	   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
	   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 445

		Inner Exception
		---------------
		Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
		Message : Login failed for user 'super'.
		Source : .Net SqlClient Data Provider
		Help link : 
		Errors : System.Data.SqlClient.SqlErrorCollection
		ClientConnectionId : c7e38451-7b62-4f2d-ad7a-ea3e17db6f9e
		Class : 14
		LineNumber : 65536
		Number : 18456
		Procedure : 
		Server : 192.255.51.85\NGSQL08QA
		State : 1
		ErrorCode : -2146232060
		Data : System.Collections.ListDictionaryInternal
		TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
		HResult : -2146232060
		Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
		   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
		   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.Open()
		   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:29:29 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:29:29 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:29:29 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 10812; Name = )
  Extended Properties: 

12/7/2018 3:30:01 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 25640; Name = )
  Extended Properties: 

12/7/2018 3:30:01 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 25640; Name = )
  Extended Properties: 

12/7/2018 3:30:01 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 25640; Name = )
  Extended Properties: 

12/7/2018 3:30:06 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 25640; Name = )
  Extended Properties: 

12/7/2018 3:33:08 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:33:08 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:33:08 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:33:12 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:34:07 PM AVR Service could not connect to AVR Service Database. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 29564; Name = )
  Extended Properties: Exception - An exception of type 'eidss.model.AVR.DataBase.AvrDataException' occurred and was caught.
-----------------------------------------------------------------------------------------
12/07/2018 15:34:07
Type : eidss.model.AVR.DataBase.AvrDataException, eidss.core, Version=6.1.0.42, Culture=neutral, PublicKeyToken=null
Message : AVR Service could not connect to AVR Service Database.
Source : eidss.avr.service
Help link : 
Data : System.Collections.ListDictionaryInternal
TargetSite : eidss.model.AVR.DataBase.DatabaseNames GetDatabaseNames()
HResult : -2146232832
Stack Trace :    at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 479
   at EIDSS.AVR.Service.WcfFacade.AVRFacade.GetDatabaseName() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AVRFacade.cs:line 528

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:34:07 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
	Message : Login failed for user '{0}'.
	Source : BLToolkit.4
	Help link : 
	Number : 18456
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
	HResult : -2146233087
	Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
	   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
	   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
	   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 473

		Inner Exception
		---------------
		Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
		Message : Login failed for user '{0}'.
		Source : .Net SqlClient Data Provider
		Help link : 
		Errors : System.Data.SqlClient.SqlErrorCollection
		ClientConnectionId : cc65d3b5-643d-4151-a00a-1f3444a75733
		Class : 14
		LineNumber : 65536
		Number : 18456
		Procedure : 
		Server : 192.255.51.85\NGSQL08QA
		State : 1
		ErrorCode : -2146232060
		Data : System.Collections.ListDictionaryInternal
		TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
		HResult : -2146232060
		Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
		   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
		   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.Open()
		   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:34:07 PM Login failed for user '{0}'. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 29564; Name = )
  Extended Properties: Exception - An exception of type 'BLToolkit.Data.DataException' occurred and was caught.
----------------------------------------------------------------------------
12/07/2018 15:34:07
Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
Message : Login failed for user '{0}'.
Source : BLToolkit.4
Help link : 
Number : 18456
Data : System.Collections.ListDictionaryInternal
TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
HResult : -2146233087
Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 473

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:34:07 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
	Message : Login failed for user '{0}'.
	Source : .Net SqlClient Data Provider
	Help link : 
	Errors : System.Data.SqlClient.SqlErrorCollection
	ClientConnectionId : cc65d3b5-643d-4151-a00a-1f3444a75733
	Class : 14
	LineNumber : 65536
	Number : 18456
	Procedure : 
	Server : 192.255.51.85\NGSQL08QA
	State : 1
	ErrorCode : -2146232060
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
	HResult : -2146232060
	Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
	   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
	   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
	   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
	   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
	   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
	   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
	   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
	   at System.Data.SqlClient.SqlConnection.Open()
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:34:07 PM Login failed for user '{0}'. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 29564; Name = )
  Extended Properties: Exception - An exception of type 'System.Data.SqlClient.SqlException' occurred and was caught.
----------------------------------------------------------------------------------
12/07/2018 15:34:07
Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
Message : Login failed for user '{0}'.
Source : .Net SqlClient Data Provider
Help link : 
Errors : System.Data.SqlClient.SqlErrorCollection
ClientConnectionId : cc65d3b5-643d-4151-a00a-1f3444a75733
Class : 14
LineNumber : 65536
Number : 18456
Procedure : 
Server : 192.255.51.85\NGSQL08QA
State : 1
ErrorCode : -2146232060
Data : System.Collections.ListDictionaryInternal
TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
HResult : -2146232060
Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:34:07 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing


12/7/2018 3:34:07 PM AVR Service could not connect to AVR Service Database. (EIDSS.AVR  Error) 
  Message: AVR Service could not connect to AVR Service Database. 
  Thread: (ID = 29564; Name = )
  Extended Properties: Exception - An exception of type 'eidss.model.AVR.DataBase.AvrDataException' occurred and was caught.
-----------------------------------------------------------------------------------------
12/07/2018 15:34:07
Type : eidss.model.AVR.DataBase.AvrDataException, eidss.core, Version=6.1.0.42, Culture=neutral, PublicKeyToken=null
Message : AVR Service could not connect to AVR Service Database.
Source : eidss.avr.service
Help link : 
Data : System.Collections.ListDictionaryInternal
TargetSite : eidss.model.AVR.DataBase.DatabaseNames GetDatabaseNames()
HResult : -2146232832
Stack Trace :    at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 479
   at EIDSS.AVR.Service.WcfFacade.AVRFacade.GetDatabaseName() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AVRFacade.cs:line 533
   at EIDSS.AVR.Service.WcfService.AvrHostKeeper.InitServiceHost() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfService\AvrHostKeeper.cs:line 57

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:34:07 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
	Message : Login failed for user '{0}'.
	Source : BLToolkit.4
	Help link : 
	Number : 18456
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
	HResult : -2146233087
	Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
	   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
	   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
	   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 473

		Inner Exception
		---------------
		Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
		Message : Login failed for user '{0}'.
		Source : .Net SqlClient Data Provider
		Help link : 
		Errors : System.Data.SqlClient.SqlErrorCollection
		ClientConnectionId : cc65d3b5-643d-4151-a00a-1f3444a75733
		Class : 14
		LineNumber : 65536
		Number : 18456
		Procedure : 
		Server : 192.255.51.85\NGSQL08QA
		State : 1
		ErrorCode : -2146232060
		Data : System.Collections.ListDictionaryInternal
		TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
		HResult : -2146232060
		Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
		   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
		   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.Open()
		   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:34:07 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:34:07 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:34:07 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 29564; Name = )
  Extended Properties: 

12/7/2018 3:35:18 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:35:18 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:35:18 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:35:22 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:36:04 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Service use 
                    EIDSS Database 'EIDSS_TH'
                    EIDSS Archive Database ''
                    AVR Database 'EIDSS_TH_AVR_Service' 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:36:05 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:36:05 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:36:05 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 1216; Name = )
  Extended Properties: 

12/7/2018 3:37:00 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:00 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:00 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:00 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:00 PM AVR Service could not connect to AVR Service Database. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 17624; Name = )
  Extended Properties: Exception - An exception of type 'eidss.model.AVR.DataBase.AvrDataException' occurred and was caught.
-----------------------------------------------------------------------------------------
12/07/2018 15:37:00
Type : eidss.model.AVR.DataBase.AvrDataException, eidss.core, Version=6.1.0.42, Culture=neutral, PublicKeyToken=null
Message : AVR Service could not connect to AVR Service Database.
Source : eidss.avr.service
Help link : 
Data : System.Collections.ListDictionaryInternal
TargetSite : eidss.model.AVR.DataBase.DatabaseNames GetDatabaseNames()
HResult : -2146232832
Stack Trace :    at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 479
   at EIDSS.AVR.Service.WcfFacade.AVRFacade.GetDatabaseName() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AVRFacade.cs:line 528

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:37:00 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
	Message : Login failed for user '{0}'.
	Source : BLToolkit.4
	Help link : 
	Number : 18456
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
	HResult : -2146233087
	Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
	   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
	   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
	   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 473

		Inner Exception
		---------------
		Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
		Message : Login failed for user '{0}'.
		Source : .Net SqlClient Data Provider
		Help link : 
		Errors : System.Data.SqlClient.SqlErrorCollection
		ClientConnectionId : 33972f30-31ab-4e7f-8f5c-729e838293e1
		Class : 14
		LineNumber : 65536
		Number : 18456
		Procedure : 
		Server : 192.255.51.85\NGSQL08QA
		State : 1
		ErrorCode : -2146232060
		Data : System.Collections.ListDictionaryInternal
		TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
		HResult : -2146232060
		Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
		   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
		   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.Open()
		   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:37:00 PM Login failed for user '{0}'. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 17624; Name = )
  Extended Properties: Exception - An exception of type 'BLToolkit.Data.DataException' occurred and was caught.
----------------------------------------------------------------------------
12/07/2018 15:37:00
Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
Message : Login failed for user '{0}'.
Source : BLToolkit.4
Help link : 
Number : 18456
Data : System.Collections.ListDictionaryInternal
TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
HResult : -2146233087
Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 473

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:37:00 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
	Message : Login failed for user '{0}'.
	Source : .Net SqlClient Data Provider
	Help link : 
	Errors : System.Data.SqlClient.SqlErrorCollection
	ClientConnectionId : 33972f30-31ab-4e7f-8f5c-729e838293e1
	Class : 14
	LineNumber : 65536
	Number : 18456
	Procedure : 
	Server : 192.255.51.85\NGSQL08QA
	State : 1
	ErrorCode : -2146232060
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
	HResult : -2146232060
	Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
	   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
	   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
	   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
	   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
	   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
	   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
	   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
	   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
	   at System.Data.SqlClient.SqlConnection.Open()
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:37:00 PM Login failed for user '{0}'. (EIDSS.AVR  Error) 
  Message: [EIDSS AVR Service version 6] WCF Service Call - Error while executing method GetDatabaseName() 
  Thread: (ID = 17624; Name = )
  Extended Properties: Exception - An exception of type 'System.Data.SqlClient.SqlException' occurred and was caught.
----------------------------------------------------------------------------------
12/07/2018 15:37:00
Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
Message : Login failed for user '{0}'.
Source : .Net SqlClient Data Provider
Help link : 
Errors : System.Data.SqlClient.SqlErrorCollection
ClientConnectionId : 33972f30-31ab-4e7f-8f5c-729e838293e1
Class : 14
LineNumber : 65536
Number : 18456
Procedure : 
Server : 192.255.51.85\NGSQL08QA
State : 1
ErrorCode : -2146232060
Data : System.Collections.ListDictionaryInternal
TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
HResult : -2146232060
Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:37:00 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing


12/7/2018 3:37:00 PM AVR Service could not connect to AVR Service Database. (EIDSS.AVR  Error) 
  Message: AVR Service could not connect to AVR Service Database. 
  Thread: (ID = 17624; Name = )
  Extended Properties: Exception - An exception of type 'eidss.model.AVR.DataBase.AvrDataException' occurred and was caught.
-----------------------------------------------------------------------------------------
12/07/2018 15:37:00
Type : eidss.model.AVR.DataBase.AvrDataException, eidss.core, Version=6.1.0.42, Culture=neutral, PublicKeyToken=null
Message : AVR Service could not connect to AVR Service Database.
Source : eidss.avr.service
Help link : 
Data : System.Collections.ListDictionaryInternal
TargetSite : eidss.model.AVR.DataBase.DatabaseNames GetDatabaseNames()
HResult : -2146232832
Stack Trace :    at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 479
   at EIDSS.AVR.Service.WcfFacade.AVRFacade.GetDatabaseName() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AVRFacade.cs:line 533
   at EIDSS.AVR.Service.WcfService.AvrHostKeeper.InitServiceHost() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfService\AvrHostKeeper.cs:line 57

Additional Info:

MachineName : DESKTOP-I7GLVAK
TimeStamp : 12/7/2018 8:37:00 PM
FullName : Microsoft.Practices.EnterpriseLibrary.ExceptionHandling, Version=5.0.414.0, Culture=neutral, PublicKeyToken=null
AppDomainName : eidss.avr.service.exe
ThreadIdentity : 
WindowsIdentity : DESKTOP-I7GLVAK\LDKing
	Inner Exception
	---------------
	Type : BLToolkit.Data.DataException, BLToolkit.4, Version=3.2.0.0, Culture=neutral, PublicKeyToken=b1d3f6ab3aadaca3
	Message : Login failed for user '{0}'.
	Source : BLToolkit.4
	Help link : 
	Number : 18456
	Data : System.Collections.ListDictionaryInternal
	TargetSite : Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
	HResult : -2146233087
	Stack Trace :    at BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 577
	   at BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4366
	   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4332
	   at BLToolkit.Data.DbManager.OpenConnection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 202
	   at BLToolkit.Data.DbManager.get_Connection() in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 177
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseName(DbManagerProxy manager) in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 487
	   at EIDSS.AVR.Service.WcfFacade.AvrDbHelper.GetDatabaseNames() in C:\NGDev\EIDSS61\EIDSS.AVR.Service\WcfFacade\AvrDbHelper.cs:line 473

		Inner Exception
		---------------
		Type : System.Data.SqlClient.SqlException, System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
		Message : Login failed for user '{0}'.
		Source : .Net SqlClient Data Provider
		Help link : 
		Errors : System.Data.SqlClient.SqlErrorCollection
		ClientConnectionId : 33972f30-31ab-4e7f-8f5c-729e838293e1
		Class : 14
		LineNumber : 65536
		Number : 18456
		Procedure : 
		Server : 192.255.51.85\NGSQL08QA
		State : 1
		ErrorCode : -2146232060
		Data : System.Collections.ListDictionaryInternal
		TargetSite : Void .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)
		HResult : -2146232060
		Stack Trace :    at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
		   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
		   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
		   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
		   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
		   at System.Data.SqlClient.SqlConnection.Open()
		   at BLToolkit.Data.DbManager.ExecuteOperation(OperationType operationType, Action operation) in C:\NGDev\EIDSS61\bltoolkit.3.2.dev\Data\DbManager.cs:line 4327



12/7/2018 3:37:01 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:01 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:01 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 17624; Name = )
  Extended Properties: 

12/7/2018 3:37:25 PM WCF Service Host (EIDSS.AVR  Verbose) 
  Message: EIDSS core initialized. 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:25 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Configured at URL http://localhost:8071/ 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:25 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Scheduler started with configuration: 
schedulerEnabled=False, 
                        timeOfSchedulerRuns=02:00:00, 
                        immediatelyRunSchedulerWhenServiceStarts=False, 
                        secondsBetweenSchedulerTasks=360 sec  
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:25 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:25 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: AVR Service use 
                    EIDSS Database 'EIDSS_TH'
                    EIDSS Archive Database ''
                    AVR Database 'EIDSS_TH_AVR_Service' 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:26 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Service Host Opened 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:26 PM WCF Service Host (EIDSS.AVR  Information) 
  Message: Scheduler started. 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:37:26 PM EIDSS Windows Service (EIDSS.AVR  Information) 
  Message: Press Enter to stop Service. 
  Thread: (ID = 35424; Name = )
  Extended Properties: 

12/7/2018 3:52:52 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetServiceVersion() 
  Thread: (ID = 32676; Name = )
  Extended Properties: 

12/7/2018 3:52:52 PM [EIDSS AVR Service version 6] WCF Service Call (EIDSS.AVR  Verbose) 
  Message: GetDatabaseName() 
  Thread: (ID = 32676; Name = )
  Extended Properties: 

